src/core/display/Transform.js

```diff
@@ -156,7 +148,6 @@ export default class Transform extends TransformBase
     set rotation(value)
     {
         this._rotation = value;
-        this._sr = Math.sin(value);
-        this._cr = Math.cos(value);
+        this.updateSkew();
     }
```

Comment:
@bigtimebuddy I don't want to change that right now, because I'm gonna change it in next iteration anyway. "updateSkew" is as good as "updateRotation", because it uses both params